test/integration/test-amp-ad-3p.js

```diff
@@ -147,6 +163,16 @@ function createIframeWithApis(fixture) {
     return poll('wait for new IO entry', () => {
       return lastIO != null;
     });
+  }).then(() => {
```

Comment:
how about inlining the return as part of the fat arrow function? :)

```javascript
.then(() => new Promise(resolve, reject) => {
  // ...
}));

Nit: `Invalid getHtml result` would be a better message.

Reply:
done